stencil-library/src/components/business-form/business-form.tsx

```diff
@@ -22,33 +22,39 @@ export class BusinessForm {
   }
 
   sendData() {
-    console.log('sendData')
+    // send data
   }
 
   validateAndSubmit(event: any) {
     event.preventDefault();
-    this.formController.validateAndSubmit(this.sendData)
+    this.formController.validateAndSubmit(this.sendData);
   }
 
   render() {
     return (
       <Host exportparts="label,input,input-invalid">
         <h1>Business Information</h1>
-        <form onSubmit={(event) => this.validateAndSubmit(event)}>
-          <div class="row gy-3">
+        <form onSubmit={event => this.validateAndSubmit(event)} class="container">
```

Comment:
Does the `container` class add margins to the right and left of the form? If so, I think we should leave that to the implementer to put embed the form in a container that has the dimensions and margins they prefer

Reply:
Hmmm I don't know, I think this change got in when fixing merging conflicts. I'll remove that.